### PR TITLE
Course Page - thumbnail preview an course image preview fix

### PR DESCRIPTION
**Pull request by kdelosreyes01**
added override file for pagethumbnail and added class to fields for page properties

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -686,13 +686,14 @@
                                         <items jcr:primaryType="nt:unstructured">
                                             <thumbnail
                                                 jcr:primaryType="nt:unstructured"
+                                                class="page-thumbnail-image"
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/_cq_dialog/.content.xml]: I think you should use the sling:resourceType="taylorsuniversity/components/content/courseimagepreview" here as well.

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/courseimagepreview/courseimagepreview.jsp [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+    private String getThumbnailUrl(Page page, ResourceResolver resourceResolver) {
+        String pagePath = page.getPath();
+
+        String url = Text.escapePath(pagePath) + ".thumb.319.319.png";
+
+
+        ValueMap pageProps = page.getProperties();
+        Calendar pageLastMod = pageProps.get("cq:lastModified", Calendar.class);
+        ValueMap imageProps = page.getProperties("course");
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/courseimagepreview/courseimagepreview.jsp]: Please determine about the "Thumbnail" or "Course" tab. And based on that get it's file-reference property.

Comment (kdelosreyes01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/courseimagepreview/courseimagepreview.jsp]: this is only for course tab image preview and should not affect the thumbnail of page by default

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/_cq_dialog/.content.xml (hunk 1 above)
Comment (kdelosreyes01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/_cq_dialog/.content.xml]: the thumbnail page preview still pertains to "/libs/cq/gui/components/coral/common/wcm/pagethumbnail" and should not be change. I've only added class here for unique class for both thumbnail page preview and course image preview

**Pull request merged**